src/core/manager.rs

```diff
@@ -709,7 +739,8 @@ impl<'a> WindowManager<'a> {
             Selector::Index(i) => self
                 .workspaces
                 .get(self.active_ws_index())
-                .and_then(|ws| ws.iter().nth(*i).and_then(|id| self.client_map.get(id))),
```

Comment:
assuming this is due to cargo fmt? Not sure why this didn't reformat for me... (I suspect it'll flip back next time I touch this file)

Reply:
The `and_then` was nested inside the other `and_then`. I don't think cargo fmt would change that. I just thought it would look more neat to chain them.

Reply:
Ah ok, that make more sense. In that case are you able to move this part of the diff to its own commit? Otherwise in got history it looks like it's part of the work to introduce the new methods which can be a bit confusing when looking back at things :slightly_smiling:

Reply:
Okay, I'll do that. I have a bad habit of throwing a full day of work in a single commit :)